tests/e2e/pageobjects/openshift/OcpApplicationPage.ts

```diff
@@ -0,0 +1,39 @@
+/*********************************************************************
+ * Copyright (c) 2019-2023 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+import 'reflect-metadata';
+import { inject, injectable } from 'inversify';
+import { DriverHelper } from '../../utils/DriverHelper';
+import { CLASSES } from '../../configs/inversify.types';
+import { By } from 'selenium-webdriver';
+import { Logger } from '../../utils/Logger';
+import { TimeoutConstants } from '../../constants/TimeoutConstants';
+import { BrowserTabsUtil } from '../../utils/BrowserTabsUtil';
+
+@injectable()
+export class OcpApplicationPage {
+
+    constructor(
+        @inject(CLASSES.DriverHelper) private readonly driverHelper: DriverHelper,
+        @inject(CLASSES.BrowserTabsUtil) private readonly browserTabsUtil: BrowserTabsUtil) {
+    }
+
+    async waitApplicationIcon(): Promise<void> {
+        Logger.debug(`${this.constructor.name}.${this.waitApplicationIcon.name}`);
+
+        await this.driverHelper.waitPresence(By.xpath('//*[@data-test-id="base-node-handler"]'), TimeoutConstants.TS_SELENIUM_LOAD_PAGE_TIMEOUT);
```

Comment:
It would improve readability of code if you extract hardcoded xpaths as a constant with names which reflects UI element, like in the next example: https://github.com/eclipse/che/pull/22181/files#r1179648473

Reply:
was inspired by:
![image](https://user-images.githubusercontent.com/72726250/235146475-1ac8ddd8-c16c-46f1-b467-9d8fe5b43dc3.png)

Reply:
https://github.com/eclipse/che/issues/22184

Reply:
There is another example of agreed declaration of locators in Che E2E TypeScript tests: https://github.com/eclipse/che/blob/main/tests/e2e/pageobjects/dashboard/Dashboard.ts#L22-L37

Reply:
Okay, I will fix it in my new classes but we should refactor it in all project to have one style everywhere

Reply:
Btw, maybe it will be better to use already locators, not stings?

Reply:
Like `private static readonly MAIN_PAGE_HEADER_LOCATOR: By = By.id('page-main-header');`

Reply:
I like an idea to use locators.

Reply:
fixed

Reply:
thanks